scapy/automaton.py

```diff
@@ -873,9 +873,20 @@ def master_filter(self, pkt):
         # type: (Packet) -> bool
         return True
 
-    def my_send(self, pkt):
-        # type: (Packet) -> None
-        self.send_sock.send(pkt)
+    def my_send(self, pkt, iface=None):
+        # type: (Packet, Optional[Union[str, list[str]]]) -> None
+        if iface is None and len(self.send_socks) == 1:
```

Comment:
What happens when `iface is None` but `len(self.send_socks) != 1`?
I'd rather have this more readable:
```Python
        if iface is None:
            if len(self.send_socks) == 1:
                self.send_socks[0].send(pkt)
            else:
                [...]
```

What do you think?

Reply:
It could be changed to 
```python
for sock in self.send_socks:
    ...
```
but now I'm looking for the `send_sock` or `send_socks` to learn more about its usage in the sources, and cannot find any mention of it. What type is it?

Reply:
This PR creates `send_socks`, so you are probably the best person to tell what type it is ;-)

`self.send_sock` is supposed to be a `SuperSocket` instance (or something that behaves the same).

Reply:
I've updated the code